### PR TITLE
Update contribute.rst

### PR DESCRIPTION
**Pull request by JulianGro**
This PR adds info about contributing to the project, trying to explain what people can do and that there is tons of work outside of the actual coding.

I do think that it look quite dry, but I also don't really have any idea how to spice it up.

A preview is available here: https://data.moto9000.moe/sphinx/%2378/contribute.html (4fe36dc)

### CODE REVIEW

File: docs/source/contribute.rst [hunk 1]
```diff
@@ -2,12 +2,64 @@
 Contribute
 ##########
 
-Vircadia is an open-source project, so all of our code and documentation is available for you to look at. You too can contribute to our endeavor for the metaverse! 
+Vircadia is an open-source project, so all of our code and documentation is available for you to look at. You too can contribute to our endeavor for the metaverse!
+Our main `GitHub repository <https://github.com/kasenvr/project-athena>`_ contains the source to many of the components of our software for creating virtual worlds.
+There is tons of ways you can contribute to the various parts that make the Vircadia project.
```
Comment (ctrlaltdavid) [inline docs/source/contribute.rst:7]:
```suggestion
There are tons of ways you can contribute to the various parts that make up Vircadia.
```

File: docs/source/contribute.rst [hunk 2]
```diff
@@ -2,12 +2,64 @@
...
+.. contents:: On This Page
+    :depth: 2
 
------------------
-Developer Section
------------------
+-------------
+Documentation
+-------------
+You can help creating and maintaining the documentation in several ways:
```
Comment (ctrlaltdavid) [inline docs/source/contribute.rst:15]:
```suggestion
You can help create and maintain the documentation in several ways:
```

File: docs/source/contribute.rst [hunk 3]
```diff
@@ -2,12 +2,64 @@
...
 
-All information required to build from source and develop the project is found in the :doc:`Developer Section <developer>`.
\ No newline at end of file
+- Report errors or missing information to the `Github issue tracker <https://github.com/kasenvr/vircadia-docs-sphinx/issues>`_.
+- Create or change documentation pages and open a `PR (pull request) on Github <https://github.com/kasenvr/vircadia-docs-sphinx/pulls>`_.
+- `Review changes <https://github.com/kasenvr/vircadia-docs-sphinx/pulls?q=is%3Apr+is%3Aopen+label%3A"needs+review">`_ to the documentation on Github. Currently we would like to have two reviews for most changes, as there is always things that the writer and the first reviewer overlooked.
```
Comment (ctrlaltdavid) [inline docs/source/contribute.rst:19]:
```suggestion
- `Review changes <https://github.com/kasenvr/vircadia-docs-sphinx/pulls?q=is%3Apr+is%3Aopen+label%3A"needs+review">`_ to the documentation on Github. We would like to have two reviewers for most changes, as there are always things that the writer and the first reviewer overlook.
```

File: docs/source/contribute.rst [hunk 4]
```diff
@@ -2,12 +2,64 @@
...
+
+
+-----------
+Translation
+-----------
+As an international project, Vircadia relies on translators to be able to reach users that do not speak english.
```
Comment (ctrlaltdavid) [inline docs/source/contribute.rst:25]:
```suggestion
As an international project, Vircadia relies on translators to be able to reach users that do not speak English.
```

File: docs/source/contribute.rst [hunk 5]
```diff
@@ -2,12 +2,64 @@
...
+Currently the only component that has translation support is the documentation.
+You can help translating on our online translation system `Weblate <https://weblate.vircadia.dev>`_ by either submitting translations or becoming a maintainer for your language and checking translations before they make it into the documentation.
```
Comment (ctrlaltdavid) [inline docs/source/contribute.rst:27]:
```suggestion
You can help translating using our online translation system `Weblate <https://weblate.vircadia.dev>`_ by either submitting translations or becoming a maintainer for your language and checking translations before they make it into the documentation.
```

File: docs/source/contribute.rst [hunk 6]
```diff
@@ -2,12 +2,64 @@
...
+More information on translating the documentation can be found in its `README <https://github.com/kasenvr/vircadia-docs-sphinx/blob/master/README.md>`_.
+
+---------
+Donations
+---------
+Most, if not all, of us work voluntarily without any payment.
+While we do not have an entity to manage donations, you can donate to specific members of the team via various services like PayPal or Patreon.
```
Comment (ctrlaltdavid) [inline docs/source/contribute.rst:34]:
```suggestion
While we do not have an entity to manage donations, you can donate to specific members of the team via various services like PayPal, Patreon, or GitHub Sponsors.
```

File: docs/source/contribute.rst [hunk 7]
```diff
@@ -2,12 +2,64 @@
...
+
+You can also put bounties on specific features you want to see implemented or issues you want to see fixed, by leaving a comment on our `issue tracker <https://github.com/kasenvr/project-athena>`_.
```
Comment (ctrlaltdavid) [inline docs/source/contribute.rst:36]:
```suggestion
You can also put bounties on specific features you want to see implemented or issues you want to see fixed, by leaving a comment on our `issue tracker <https://github.com/kasenvr/project-athena/issues>`_.
```

File: docs/source/contribute.rst [hunk 8]
```diff
@@ -2,12 +2,64 @@
...
+Of course bounties are also an option for other parts of the project like the documentation.
```
Comment (ctrlaltdavid) [inline docs/source/contribute.rst:37]:
```suggestion
Of course, bounties are also an option for other parts of the project like the documentation.
```

File: docs/source/contribute.rst [hunk 9]
```diff
@@ -2,12 +2,64 @@
...
+The bounties are based on trust, so if you are unsure about something speak to a core developer or bring it up at a community or developer meeting.
+Most of the developers are highly trained and experienced, normally earning upwards of 40$ an hour on the free market. Keep that in mind if you want to see something done that no one is personally interested in.
```
Comment (ctrlaltdavid) [inline docs/source/contribute.rst:39]:
```suggestion
Most of the developers are highly trained and experienced, normally earning upwards of 40$ an hour on the free market. Keep that in mind if you want to see something done that no one else is personally interested in.
```

File: docs/source/contribute.rst [hunk 10]
```diff
@@ -2,12 +2,64 @@
...
+
+-----------
+Development
+-----------
+You can help the general development of the Vircadia project without being much a of a developer yourself by:
+
+- filing issues on our `issue tracker <https://github.com/kasenvr/project-athena>`_,
```
Comment (ctrlaltdavid) [inline docs/source/contribute.rst:46]:
```suggestion
- filing issues on our `issue tracker <https://github.com/kasenvr/project-athena/issues>`_,
```

File: docs/source/contribute.rst [hunk 11]
```diff
@@ -2,12 +2,64 @@
...
+- helping to narrow down those issues,
+- or `testing pull requests <https://github.com/kasenvr/project-athena/pulls?q=is%3Apr+is%3Aopen+label%3A"needs+QA+(testing)">`_.
+
+If you are a developer you can contribute directly to the project via pull request. Keep the `guidelines <https://github.com/kasenvr/project-athena/blob/master/CONTRIBUTING.md>`_ in mind when contributing to the main software.
```
Comment (ctrlaltdavid) [inline docs/source/contribute.rst:50]:
```suggestion
If you are a developer, you can contribute directly to the project via pull requests. Keep the `guidelines <https://github.com/kasenvr/project-athena/blob/master/CONTRIBUTING.md>`_ in mind when contributing to the main software.
```

File: docs/source/contribute.rst [hunk 12]
```diff
@@ -2,12 +2,64 @@
...
+------------------
+There is also a lot of miscellaneous work that comes up every now and then:
+
+- 2D art like posters or banners,
+- 3D models,
+- scripts,
+- video editing,
+- and so on.
+
+It is tremendously helpful for a developers to have someone they can ask when they need work outside of their area of experience done.
```
Comment (ctrlaltdavid) [inline docs/source/contribute.rst:65]:
```suggestion
It is tremendously helpful for a developers to have someone they can ask when they need work done outside of their area of experience.
```

**Review by ctrlaltdavid — CHANGES_REQUESTED** [review]

**Comment by JulianGro** [comment]
@ctrlaltdavid I took all your suggestions. Keep in mind that this was a draft, since I still hadn't decided if I wanted to add more pages and I hadn't proof read it myself yet.

**Review by ctrlaltdavid — APPROVED** [review]

File: docs/source/contribute.rst [hunk 13]
```diff
@@ -2,12 +2,64 @@
...
+
+You can also put bounties on specific features you want to see implemented or issues you want to see fixed, by leaving a comment on our `issue tracker <https://github.com/kasenvr/project-athena/issues>`_.
+Of course, bounties are also an option for other parts of the project like the documentation.
+The bounties are based on trust, so if you are unsure about something speak to a core developer or bring it up at a community or developer meeting.
+Most of the developers are highly trained and experienced, normally earning upwards of 40$ an hour on the free market. Keep that in mind if you want to see something done that no one else is personally interested in.
+
+-----------
+Development
+-----------
+You can help the general development of the Vircadia project without being much a of a developer yourself by:
```
Comment (kasenvr) [inline docs/source/contribute.rst:44]:
```suggestion
You can help the general development of the Vircadia project without being a developer yourself by:
```

File: docs/source/contribute.rst [hunk 14]
```diff
@@ -2,12 +2,64 @@
...
+------------------
+There is also a lot of miscellaneous work that comes up every now and then:
+
+- 2D art like posters or banners,
+- 3D models,
+- scripts,
+- video editing,
+- and so on.
+
+It is tremendously helpful for a developers to have someone they can ask when they need work done outside of their area of experience.
```
Comment (kasenvr) [inline docs/source/contribute.rst:65]:
```suggestion
It is tremendously helpful for developers to have someone that they can ask for assistance when they need work done that is outside their area of expertise.
```

File: docs/source/contribute.rst [hunk 15]
```diff
@@ -2,12 +2,64 @@
...
+- filing issues on our `issue tracker <https://github.com/kasenvr/project-athena/issues>`_,
+- helping to narrow down those issues,
+- or `testing pull requests <https://github.com/kasenvr/project-athena/pulls?q=is%3Apr+is%3Aopen+label%3A"needs+QA+(testing)">`_.
+
+If you are a developer, you can contribute directly to the project via pull requests. Keep the `guidelines <https://github.com/kasenvr/project-athena/blob/master/CONTRIBUTING.md>`_ in mind when contributing to the main software.
+Some additional information can be found in the :doc:`Developer Section <developer>`.
+An important job for developers that have become more accustomed to the platform is `code reviews <https://github.com/kasenvr/project-athena/pulls?q=is%3Apr+is%3Aopen+label%3A"needs+CR+(code+review)">`_ of open pull requests.
+
+------------------
+Miscellaneous work
```
Comment (kasenvr) [inline docs/source/contribute.rst:55]:
```suggestion
Miscellaneous Work
```

File: docs/source/contribute.rst [hunk 16]
```diff
@@ -2,12 +2,64 @@
...
+
+---------
+Donations
+---------
+Most, if not all, of us work voluntarily without any payment.
+While we do not have an entity to manage donations, you can donate to specific members of the team via various services like PayPal, Patreon, or Github Sponsors.
+
+You can also put bounties on specific features you want to see implemented or issues you want to see fixed, by leaving a comment on our `issue tracker <https://github.com/kasenvr/project-athena/issues>`_.
+Of course, bounties are also an option for other parts of the project like the documentation.
+The bounties are based on trust, so if you are unsure about something speak to a core developer or bring it up at a community or developer meeting.
```
Comment (kasenvr) [inline docs/source/contribute.rst:38]:
```suggestion
Bounties are created and assigned based on trust, so if you are unsure about something regarding one then speak to a core developer or bring it up at a community or developer meeting.
```

File: docs/source/contribute.rst [hunk 17]
```diff
@@ -2,12 +2,64 @@
...
+Most of the developers are highly trained and experienced, normally earning upwards of 40$ an hour on the free market. Keep that in mind if you want to see something done that no one else is personally interested in.
```
Comment (kasenvr) [inline docs/source/contribute.rst:39]:
```suggestion
Most of the developers are highly trained and experienced, normally earning upwards of $40 an hour in the free market. Keep that in mind if you want to see something accomplished that no one else is personally interested in.
```

File: docs/source/contribute.rst [hunk 18]
```diff
@@ -2,12 +2,64 @@
...
+-----------
+As an international project, Vircadia relies on translators to be able to reach users that do not speak English.
+Currently the only component that has translation support is the documentation.
+You can help translating using our online translation system `Weblate <https://weblate.vircadia.dev>`_ by either submitting translations or becoming a maintainer for your language and checking translations before they make it into the documentation.
+More information on translating the documentation can be found in its `README <https://github.com/kasenvr/vircadia-docs-sphinx/blob/master/README.md>`_.
+
+---------
+Donations
+---------
+Most, if not all, of us work voluntarily without any payment.
```
Comment (kasenvr) [inline docs/source/contribute.rst:33]:
```suggestion
Vircadia is primarily driven by the voluntary contributions of its community and core team.
```

File: docs/source/contribute.rst [hunk 19]
```diff
@@ -2,12 +2,64 @@
 Contribute
 ##########
 
-Vircadia is an open-source project, so all of our code and documentation is available for you to look at. You too can contribute to our endeavor for the metaverse! 
+Vircadia is an open-source project, so all of our code and documentation is available for you to look at. You too can contribute to our endeavor for the metaverse!
+Our main `GitHub repository <https://github.com/kasenvr/project-athena>`_ contains the source to many of the components of our software for creating virtual worlds.
+There are tons of ways you can contribute to the various parts that make up Vircadia.
```
Comment (kasenvr) [inline docs/source/contribute.rst:7]:
```suggestion
There are plenty of ways you can contribute to the various parts that make up Vircadia.
```

File: docs/source/contribute.rst [hunk 20]
```diff
@@ -2,12 +2,64 @@
 Contribute
 ##########
 
-Vircadia is an open-source project, so all of our code and documentation is available for you to look at. You too can contribute to our endeavor for the metaverse! 
+Vircadia is an open-source project, so all of our code and documentation is available for you to look at. You too can contribute to our endeavor for the metaverse!
+Our main `GitHub repository <https://github.com/kasenvr/project-athena>`_ contains the source to many of the components of our software for creating virtual worlds.
```
Comment (kasenvr) [inline docs/source/contribute.rst:6]:
```suggestion
Our main `git repository <https://github.com/kasenvr/project-athena>`_ contains the source to many of the components of our software for creating virtual worlds.
```

Comment (kasenvr) [reply docs/source/contribute.rst:6]: The reason for this is to keep agnostic what service is hosting the overall git repos as that can change from GitHub at anytime. Whereas things like "Issues" are specific to GitHub potentially and the wording may have to change if it changes from GitHub anyway.

File: docs/source/contribute.rst [hunk 21]
```diff
@@ -2,12 +2,64 @@
...
-Developer Section
------------------
+-------------
+Documentation
+-------------
+You can help create and maintain the documentation in several ways:
 
-All information required to build from source and develop the project is found in the :doc:`Developer Section <developer>`.
\ No newline at end of file
+- Report errors or missing information to the `Github issue tracker <https://github.com/kasenvr/vircadia-docs-sphinx/issues>`_.
```
Comment (kasenvr) [inline docs/source/contribute.rst:17]:
```suggestion
- Report errors or missing information to the `GitHub issue tracker <https://github.com/kasenvr/vircadia-docs-sphinx/issues>`_.
```

File: docs/source/contribute.rst [hunk 22]
```diff
@@ -2,12 +2,64 @@
...
+- Create or change documentation pages and open a `PR (pull request) on Github <https://github.com/kasenvr/vircadia-docs-sphinx/pulls>`_.
```
Comment (kasenvr) [inline docs/source/contribute.rst:18]:
```suggestion
- Create or change documentation pages and open a `PR (pull request) on GitHub <https://github.com/kasenvr/vircadia-docs-sphinx/pulls>`_.
```

**Review by kasenvr — CHANGES_REQUESTED** [review]

File: docs/source/contribute.rst [hunk 23]
```diff
@@ -2,12 +2,64 @@
...
+---------
+Donations
+---------
+Vircadia is primarily driven by the voluntary contributions of its community and core team.
+While we do not have an entity to manage donations, you can donate to specific members of the team via various services like PayPal, Patreon, or Github Sponsors.
+
+You can also put bounties on specific features you want to see implemented or issues you want to see fixed, by leaving a comment on our `issue tracker <https://github.com/kasenvr/project-athena/issues>`_.
+Of course, bounties are also an option for other parts of the project like the documentation.
+Bounties are created and assigned based on trust, so if you are unsure about something regarding one then speak to a core developer or bring it up at a community or developer meeting.
+Most of the developers are highly trained and experienced, normally earning upwards of $40 an hour in the free market. Keep that in mind if you want to see something accomplished that no one else is personally interested in.
```
Comment (kasenvr) [inline docs/source/contribute.rst:39]:
```suggestion
Most of the developers are highly trained and experienced, normally earning upwards of US$40 an hour in the free market. Keep that in mind if you want to see something accomplished that no one else is personally interested in.
```

Comment (kasenvr) [reply docs/source/contribute.rst:39]: One last thing, heh, I realized that $40 may not make sense to everyone unless specified to be USD. :)

**Review by kasenvr — APPROVED** [review]

**Pull request merged**